src/shared/ui/Select/OptionList.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+import React from 'react';
+
+import { useSelectContext } from './Select.context';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export function OptionList({ children }: Props) {
+  const { size } = useSelectContext();
+
+  const sizeClass = size === 'md' ? 'text-sm min-w-24' : 'text-base md:text-lg';
+
+  return (
+    <div
+      className={`mt-1 max-h-60 overflow-y-auto rounded-md border border-gray-200 bg-white font-semibold text-gray-400 ${sizeClass}`}
```

Comment:
가독성, 유지보수성을 생각하면 다음과 `cn` 유틸리티를 사용하면 좋을 것 같은데 어떻게 생각하시나요? 전체적으로 사용하지 않으셨던데 이유가 있는지 궁금합니다~!

```suggestion
      className={cn(`mt-1 max-h-60 overflow-y-auto rounded-md border border-gray-200 bg-white font-semibold text-gray-400` ,sizeClass )}
```

Reply:
cn 유틸리티를 사용하지 않은 것은 미처 추가하지 못하였습니다 .. 
해당 유틸리티 모두 적용하여 가독성과 유지보수성을 높였습니다!